src/Extensions/UriExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Kros.Utils;
+using System;
+using System.Linq;
+
+namespace Kros.Extensions
+{
+    /// <summary>
+    /// Extensions for better work with URI.
+    /// </summary>
+    public static class UriExtensions
+    {
+        /// <summary>
+        /// Gets the domain name from URI.
+        /// </summary>
+        /// <param name="uri">The URI.</param>
+        /// <returns>Domain name.</returns>
+        /// <remarks>
+        /// From gmail.google.com return google.com.
+        /// </remarks>
+        public static string GetDomain(this string uri)
+            => new UriBuilder(Check.NotNullOrWhiteSpace(uri, nameof(uri))).Uri.GetDomain();
```

Comment:
Why `UriBuilder`? We can use directly `new Uri(uri, UriKind.RelativeOrAbsolute)`.

Reply:
I can't use `Uri` constructor. When relative path like `google.com` is used, then `Host` property can't be used. `InvalidOperationExcpetion` _This operation is not supported for a relative URI_.